.github/workflows/quick-test.yml

```diff
@@ -25,6 +25,24 @@ jobs:
       - name: super-test
         run: |
             ./super-test.sh quick ${{ matrix.compiler }}
+  Linux-lock-tracking:
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        compiler: [clang-9]
+        features: ["-DKJ_TRACK_LOCK_BLOCKING=1 -DKJ_SAVE_ACQUIRED_LOCK_INFO=1"]
+    steps:
+      - uses: actions/checkout@v2
+      - name: install dependencies
+        run: |
+            export DEBIAN_FRONTEND=noninteractive
+            sudo apt-get install -y build-essential git zlib1g-dev cmake libssl-dev ${{ matrix.compiler }}
+      - name: super-test
+        run: |
+            # librt is used for creating timers via (timer_X) functions so that we can simulate a deadlock
```

Comment:
I was thinking more along the lines of: "librt is needed for timer_create in the unit tests for lock tracking"

The goal is not to fully explain the reasoning, but just to give people a hint where to look if they're trying to figure out why this random extra flag is here. Sometimes less information is better. :)